cypress/support/fragments/orders/basicOrderLine.js

```diff
@@ -52,5 +56,74 @@ export default {
       instructions: '',
       vendorAccount: '1234'
     }
+  },
+  specialOrderLine: {
```

Comment:
think to define specialOrderLine based on defaultOrderLine with population of additional properties required to this special order line.

Reply:
should be fixed/discussed